To get rid of warnings:

    ocamlfind: [WARNING] Package `threads': Linking problems may arise because of the missing -thread or -vmthread switch

--- a/Makefile
+++ b/Makefile
@@ -199,7 +199,7 @@
 
 RELEASE_TAGS=$(if $(FLOW_RELEASE),-tag warn_a,)
 
-OCB=ocamlbuild -use-ocamlfind -no-links
+OCB=ocamlbuild -use-ocamlfind -no-links -tag thread
 
 all: bin/flow$(EXE)
 
